util/tlm/SConstruct

```diff
@@ -64,6 +66,12 @@ env.Append(CXXFLAGS=['-std=c++17',
 if gem5_variant == 'debug':
     env.Append(CXXFLAGS=['-g', '-DGEM5_DEBUG'])
 
+AddOption('--duplicate-sources', action='store_true', default=False,
+          dest='duplicate_sources',
+          help='Create symlinks to sources in the build directory')
```

Comment:
You don't need this one as the option is already present below (see --no-duplicate-sources). You just need to bump the option over here before it is used

Reply:
What do you mean by "bump the option over here before it is used"? What I was referencing for this code was the main gem5 SConstruct file which has the following on line 145:
```python
AddOption('--duplicate-sources', action='store_true', default=False,
          dest='duplicate_sources',
          help='Create symlinks to sources in the build directory')
AddOption('--no-duplicate-sources', action='store_false',
          dest='duplicate_sources',
          help='Do not create symlinks to sources in the build directory')
```
so I just assumed that you would always need to define both flags

Reply:
You don't need to define both options. You just need to move the following code:

```
AddOption('--no-duplicate-sources', action='store_false',
          dest='duplicate_sources',
          help='Do not create symlinks to sources in the build directory')
```

To line 69.